src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,103 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_SIZE = 10;
+    private static final double RESIZE_NUMBER = 1.5;
+    private int size;
+    private Object[] elementsData = new Object[DEFAULT_SIZE];
```

Comment:
use the parametrized type
This field should be initialized in constructor

```suggestion
    private T [] elementsData;
```

Reply:
+